config/initializers/riiif.rb

```diff
@@ -26,4 +26,5 @@ def logger
 
 # Riiif::Image.authorization_service = IIIFAuthorizationService
 
-Riiif::Engine.config.cache_duration_in_days = 365
+Riiif::Engine.config.cache_duration_in_days = 90
+Riiif::Image.file_resolver.cache_path = 'tmp/network_files'
```

Comment:
@scherztc If we keep the cache in tmp/network_files, the cache will get blown away every time we deploy.  I think we should make this directory be `/srv/apps/scholar-riiif-cache` (outside of our app directory).  Calling it scholar-riiif-cache will make it more identifiable than "network_files".